tests/common/test_transaction.py

```diff
@@ -1023,7 +1018,6 @@ def test_create_transfer_transaction_multiple_io(user_pub, user_priv,
                     'owners_before': [
                         user2_pub
                     ],
-                    'fid': 1,
                     'fulfillment': None,
                     'input': {
                         'txid': tx.id,
```

Comment:
Sure, I'm not sure what approach to take exactly but I'll have another look at the code and see if I can find any vectors... :pick: